packages/framework/attributor/src/test/attribution/sharedString.attribution.spec.ts

```diff
@@ -91,34 +89,43 @@ interface FuzzTestState extends BaseFuzzTestState {
 	serializer?: Encoder<IAttributor, string>;
 }
 
-interface ClientSpec {
+// Type aliases are used here instead of interfaces to avoid ts error 2345:
+//   Index signature for type 'string' is missing in type 'Operation[]'
+// when sent to writeJson which requires Jsonable<unknown>.
+/* eslint-disable @typescript-eslint/consistent-type-definitions */
```

Comment:
Do you understand why type vs. interface makes a difference?  (I don't, which makes me nervous.)

Reply:
I think I know. `interfaces` are assumed to be augmented but "simple" types are not. I think this the best comment in the now closed typescript issue: https://github.com/microsoft/TypeScript/issues/15300#issuecomment-657218214

What I boiled it down to in my head whether right or wrong is that with type aliases more likely we are talking about simple data AND we think we'll be getting just what we think.